packages/core/src/state-management/repository.ts

```diff
@@ -119,10 +119,10 @@ export class Repository {
     if (commit == null) {
       throw new Error(`No genesis commit found with CID ${genesisCid.toString()}`)
     }
-    // Do not check timestamp here
+    // Do not check for possible key revocation here, as we will do so later after loading the tip (or learning that the genesis commit *is* the current tip), when we will have timestamp information for when the genesis commit was anchored. In order to disable this check for key rotation, set a timestamp that will be considered earlier than any other timestamp so it will always appear to have happened earlier than any possible key revocation.
     const state = await handler.applyCommit(
       commit,
-      { cid: streamId.cid, timestamp: null },
+      { cid: streamId.cid, timestamp: 1 },
```

Comment:
Will this fail now that we check that a signature has to be made after the key was added to the DID doc using the `updated` property?

Reply:
Indeed. It will fail. Added a commit that passes `disableTimecheck` flag.